### ISSUE TITLE
Oracle docs improvements

### ISSUE THREAD

**Issue by acrewdson** [elastic/connectors#1882]
When testing the [Oracle connector](https://www.elastic.co/guide/en/enterprise-search/current/connectors-oracle.html) recently I noticed a few things that could be improved in those docs:

>  Port number of the Oracle database server. Default value is 9090. 

Seems like a strange port to specify as a default. `1521` might be a more reasonable default; also seems like `2484` is often typically used for TLS-encrypted Oracle connections, [see here](https://helpcenter.netwrix.com/bundle/Auditor_10.0/page/Content/Additional_Configuration/Configure_Ports/Configure_Ports_Oracle.htm).

>  Comma-separated list of tables to monitor for changes. Default value is *. Examples: 

When we give example table names, we show `table_1`, `table_2`. However Oracle table names are typically uppercase, and specifying them as lower case in the connectors config will cause them to not be found by the connector.

> `oracle_home`

This is confusing because in the connectors config screen the field is labeled `Path of Oracle Service`. It should probably be `Path to Oracle Home` or something along those lines. Either way it should be consistent between docs and UI.

> `wallet_configuration_path`

As with the item above, this is inconsistent between UI and docs. The UI confusingly labels it `Path of Oracle Service configuration files`. It should probably be something like `Path to Oracle SSL Wallet`. Also the text in the docs, 'Path of Oracle configuration files', feels un-idiomatic. 'Path to' would make more sense.

**Comment by leemthompo** [comment]
> Either way it should be consistent between docs and UI.

@acrewdson this is a good point, but there's one nuance here. We tend to document the values here as they appear in the `get_default_configuration` in source config. For example [`oracle_home`](https://github.com/elastic/connectors/blob/main/connectors/sources/oracle.py#L348).

We have a note about this at top of configuration section.

<img width="865" alt="Screenshot 2023-11-09 at 13 14 55" src="https://github.com/elastic/connectors/assets/32779855/c944efe3-ed65-4735-9f33-7b903ad3e703">

For _native_ connectors ([example](https://www.elastic.co/guide/en/enterprise-search/current/connectors-dropbox.html#connectors-dropbox-native-connector-reference)) we use the labels as appear in Kibana. 

BUT I'm not really sure if we need to have this distinction though, and maybe we should just document the labels in the configuration section?

I'm going to open a PR to make the other fixes and we'll see what folks have to say about the config labels topic because that impacts every connector. I can make that update separately if we agree to change the current approach.

Be interested to hear what other folks think though: cc @danajuratoni @seanstory @timgrein @navarone-feekery @jedrazb

**Comment by parthpuri-elastic** [comment]
@leemthompo, I have raised PR #1899 for updating the labels at connector level. Now, we need to update the HTML page as below.

1. We have removed the default value of port from the connector. So, we can remove the default value from the documentation.
2. We have checked for the tables; we can capitalize the example for `Comma-separated list of tables` as `TABLE_1, TABLE_2`.
3. For the labels, refer PR #1899. I have updated the label as `Path to Oracle Home` for `oracle_home` and `Path to SSL Wallet configuration files` for `wallet_configuration_path`.
